cmd/util/ledger/migrations/storage_v6.go

```diff
@@ -1295,7 +1212,7 @@ func getValue(
 		}
```

Comment:
Same for `if newValue != nil` above: Shouldn't the migration abort and not produce a dictionary where values are `nil`?